src/index.js

```diff
@@ -291,6 +291,10 @@ export class ReactNativeModal extends Component {
       this.state.pan.setValue({ x: 0, y: 0 });
     }
 
+    if (!this.contentRef) {
+      return;
+    }
+
     this.contentRef[this.animationIn](this.props.animationInTiming).then(() => {
```

Comment:
Could you please wrap the code below like this instead?
```
if (this.contentRef) {
  this.contentRef[this.animationIn](this.props.animationInTiming).then(() => {
  ...
}
```

Thanks!

Reply:
done